<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ma première UI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>188</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nomLabel">
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nomLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="prNomLabel">
       <property name="text">
        <string>prénom</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="prNomLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateTimeEdit" name="dateDateTimeEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="villeLabel">
       <property name="text">
        <string>Ville</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="villeLineEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="validButton">
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="texteResultat">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>214</y>
      <width>271</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lab">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>317</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menu_Fichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionA_Propos">
   <property name="text">
    <string>A_Propos</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
